expression-language/pom.xml

```diff
@@ -4,11 +4,11 @@
 
   <parent>
     <groupId>io.camunda</groupId>
-    <artifactId>zeebe-root</artifactId>
+    <artifactId>zeebe-parent</artifactId>
     <version>1.3.0-SNAPSHOT</version>
+    <relativePath>../parent/pom.xml</relativePath>
   </parent>
 
-  <groupId>io.camunda</groupId>
```

Comment:
:question: Is the `groupId` now properly inherited from the parent or why did you remove this here?

Reply:
Exactly :+1: It doesn't hurt to leave, but it is redundant and flagged as a warning.